ghc-lib-gen/src/Ghclibgen.hs

```diff
@@ -1314,7 +1326,7 @@ libBinParserLibModules ghcFlavor = do
 
 happyBounds :: GhcFlavor -> String
 happyBounds ghcFlavor
-  | series < GHC_9_8 = "== 1.20.*"
+  | series < GHC_9_8 = "< 1.21"
```

Comment:
What's the difference here? Is there a situation where we need to supply 1.19 to GHC9.8?

Reply:
yes, ghc-lib-ghc-9.0.2-ghc-8.10.7.yml (i have no idea why this didn't get picked up before now).